packages/favfreak/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=favfreak
+_pkgname=FavFreak
+pkgver=19.3a0bf2a
+pkgrel=1
+pkgdesc='Weaponizing favicon.ico for BugBounties , OSINT and what not.'
+arch=('any')
+groups=('blackarch' 'blackarch-recon' 'blackarch-fingerprint')
+url='https://github.com/devanshbatham/FavFreak'
+license=('custom:unknown')
```

Comment:
When no license contacting upstream is important https://github.com/devanshbatham/FavFreak/issues/6

Reply:
no update from from @devanshbatham as of yet. Will update once we have this.

Reply:
updated. Applying